packages/retryable-monitor/handlers/notion/alertUntriagedRetraybles.ts

```diff
@@ -26,7 +27,8 @@ const isNearExpiry = (iso: string | undefined, hours = 24) => {
 }
 
 export const alertUntriagedNotionRetryables = async (
-  childChains: ChildNetwork[] = []
+  childChains: ChildNetwork[] = [],
+  enableAutoRedeem = false // controls >96h silent redemption
```

Comment:
as per our app-eng discussion, wasn't `enableAutoRedeem` supposed to be `true` by default? 
If it needs an explicit flag to enable it, then we will also need to add it to the `arbitrum-token-bridge` repo

Reply:
If the enableAutoRedeem defaults to true then the `--autoRedeem` flag doesn’t really do anything anymore. I'd say let's keep the default as false, and then let folks decide on whether they want this feature to be on or not. We def need to add t to the token bridge repo though yea